app/frontend/components/autoLogout.js

```diff
@@ -0,0 +1,72 @@
+const {Component} = require('preact')
+const connect = require('unistore/preact').connect
+const actions = require('../actions')
+
+class AutoLogout extends Component {
+  constructor(props) {
+    super(props)
+    this.resetTimer = this.resetTimer.bind(this)
+    this.logout = this.logout.bind(this)
+  }
+
+  logout() {
+    this.props.closeDemoWalletWarningDialog()
+    this.props.setLogoutNotificationOpen(true)
+    this.props.logout()
+  }
+
+  resetTimer() {
+    if (this.props.walletIsLoaded) {
+      clearTimeout(this.timer)
+      this.timer = setTimeout(this.logout, 15 * 60 * 1000)
```

Comment:
this should probably be tweakable in .env by something like ADALITE_LOGOUT_AFTER and it could be the amount of seconds. If it is not set, it could be interpreted as no timeout at all.